Specs/Scene/PointCloud3DTileContentSpec.js

```diff
@@ -439,6 +441,172 @@ defineSuite([
+        });
+        var center = new Cartesian3.fromRadians(centerLongitude, centerLatitude, 5.0);
+        scene.camera.lookAt(center, new HeadingPitchRange(0.0, -1.57, 5.0));
+        scene.fxaa = false;
+        scene.camera.zoomIn(6);
+
+        return Cesium3DTilesTester.loadTileset(scene, pointCloudNoColorUrl).then(function(tileset) {
+            var noAttenuationPixelCount = 16;
+
+            // pointCloudNoColorUrl is a single tile with GeometricError = 0,
+            // which results in default baseResolution being computed
+            tileset.pointShading.attenuation = true;
+            tileset.pointShading.geometricErrorScale = 1.0;
+            tileset.pointShading.maximumAttenuation = undefined;
+            tileset.pointShading.baseResolution = CesiumMath.EPSILON20;
+            tileset.maximumScreenSpaceError = 16;
+            expect(scene).toRenderPixelCount(noAttenuationPixelCount);
+
+            scene.destroyForSpecs();
+        });
+    });
+
+    it('modulates attenuation using the geometricErrorScale parameter', function() {
+        var scene = createScene({
+            canvas : createCanvas(10, 10)
+        });
+        var center = new Cartesian3.fromRadians(centerLongitude, centerLatitude, 5.0);
+        scene.camera.lookAt(center, new HeadingPitchRange(0.0, -1.57, 5.0));
+        scene.fxaa = false;
+        scene.camera.zoomIn(6);
+
+        return Cesium3DTilesTester.loadTileset(scene, pointCloudNoColorUrl).then(function(tileset) {
+            var noAttenuationPixelCount = 16;
+
+            tileset.pointShading.attenuation = true;
+            tileset.pointShading.geometricErrorScale = 0.0;
+            tileset.pointShading.maximumAttenuation = undefined;
+            tileset.pointShading.baseResolution = undefined;
+            tileset.maximumScreenSpaceError = 1;
+            expect(scene).toRenderPixelCount(noAttenuationPixelCount);
+
+            scene.destroyForSpecs();
+        });
+    });
+
+
+    it('attenuates points based on geometric error in 2D', function() {
+        var scene = createScene({
+            canvas : createCanvas(10, 10)
+        });
+        var center = new Cartesian3.fromRadians(centerLongitude, centerLatitude, 5.0);
+        scene.camera.lookAt(center, new HeadingPitchRange(0.0, -1.57, 5.0));
+        scene.fxaa = false;
+        scene.camera.zoomIn(6);
+
+        return Cesium3DTilesTester.loadTileset(scene, pointCloudNoColorUrl).then(function(tileset) {
+
+            var noAttenuationPixelCount = 16;
+
+            scene.morphTo2D(0);
+            tileset.pointShading.attenuation = true;
+            tileset.pointShading.geometricErrorScale = 1.0;
+            tileset.pointShading.maximumAttenuation = undefined;
+            tileset.pointShading.baseResolution = undefined;
+            tileset.maximumScreenSpaceError = 16;
+            expect(scene).toRenderPixelCountGreaterThan(noAttenuationPixelCount);
+
+            scene.destroyForSpecs();
+        });
+    });
```

Comment:
Given that all these tests are really similar, there could be a common function that all these tests call. Each test would pass in a callback function that makes the change (e.g. in here, the callback would contain  `scene.morphTo2D(0);`.